install.md

```diff
@@ -18,10 +18,4 @@ oc edit route cluster-support-bot
     termination: edge
```

Comment:
We should drop this `oc edit route cluster-support-bot` bit too, now that we no longer need a Route.

Reply:
done